src/platform/user/profile/vap-svc/util/transactions.js

```diff
@@ -64,6 +64,12 @@ export const INVALID_EMAIL_ADDRESS_ERROR_CODES = new Set([
   'VET360_EMAIL305',
 ]);
 
+export const INVALID_PHONE_ERROR_CODES = new Set([
+  'VET360_PHON126', // Phone area code pattern must match "[0-9]+"
+  'VET360_PHON211', // New: Area Codes do not end with the same two digits
+  'VET360_PHON213', // New: Area Codes do not include 9 as the middle digit
+]);
```

Comment:
Do I need to include the non-`VET360_` prefixes?

Reply:
I was wondering the same thing. I don't think so, but it'd be totally safe to add those to the list since I think those codes will only appear thanks to [the change @annaswims made](https://github.com/department-of-veterans-affairs/vets-api/pull/7474) so there's no way non-`VET360_` codes would appear.